test/Interop/Cxx/reference/rvalue-reference.swift

```diff
@@ -0,0 +1,54 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -enable-experimental-cxx-interop -Onone) | %FileCheck %s
+// RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -enable-experimental-cxx-interop -O) | %FileCheck -check-prefix=CHECK-OPT %s
+//
+// REQUIRES: executable_test
+
+import SpecialMembers
+
+func consume(_ x: consuming MoveOnly) {}
+func consume(_ x: consuming Copyable) {}
+
+func main() {
```

Comment:
nit: can you break this test up into separate functions, and briefly document the expected behavior? It's a bit hard to keep track of which variables are allocated where.

Reply:
breaking it up into separate functions should also make it clearer which `Copyable` instance is being destroyed at the end.

Reply:
I split it up in 4.